src/Directory.Build.props

```diff
@@ -5,6 +5,11 @@
   <!-- This file is imported by all projects at the beginning of the project files -->
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
+  <PropertyGroup>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
PDBs are already embedded in release mode later in this file, so I just removed this line.